symengine/polys/uintpoly_flint.cpp

```diff
@@ -102,12 +97,11 @@ integer_class UIntPolyFlint::get_coeff(unsigned int x) const
     return to_integer_class(poly_.coeff(x));
 }
 
-flint::fmpzxx_srcref UIntPolyFlint::get_coeff_ref(unsigned int x) const
+fz_t UIntPolyFlint::get_coeff_ref(unsigned int x) const
```

Comment:
This change is not correct, we needed the reference to the coefficient inside the polynomial.